tools/harness-thci/OpenThread.py

```diff
@@ -401,7 +435,10 @@ def __startOpenThread(self):
                         for addr in self._addressfilterSet:
                             self.addBlockedMAC(addr)
 
-            if self.deviceRole in [Thread_Device_Role.Leader, Thread_Device_Role.Router, Thread_Device_Role.REED]:
+            if self.deviceRole in [
+                    Thread_Device_Role.Leader,
+                    Thread_Device_Role.Router,
+                    Thread_Device_Role.REED]:
                 self.__setRouterSelectionJitter(1)
 
             if self.__sendCommand('ifconfig up')[0] == 'Done':
```

Comment:
Seem to still be some places where ``[0] == 'Done'`` is used instead of ``[-1] == 'Done' for the __sendCommand. I found a bunch by searching for ``[0] == 'Done``

Reply:
Thanks for the reminder. will update to unify the usage soon.